dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeTrace.kt

```diff
@@ -15,9 +15,13 @@ internal class BridgeTrace : DdTrace {
 
     private val spanMap: MutableMap<String, Span> = mutableMapOf()
 
-    override fun startSpan(operation: String, timestamp: Long, context: Map<String, Any?>): String {
+    override fun startSpan(
+        operation: String,
+        timestampMs: Long,
```

Comment:
`timestamp` -> `timestampMs` change is needed to fix a following compilation warning:

```
BridgeTrace.kt: (18, 47): The corresponding parameter in the supertype 'DdTrace' is named 'timestampMs'. This may cause problems when calling this function with named arguments.
```